core/src/main/java/org/apache/accumulo/core/spi/compaction/CompactionDirectivesBuilder.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.spi.compaction;
+
+class CompactionDirectivesBuilder implements CompactionDirectives.Builder {
+
+  private CompactionServiceId service;
```

Comment:
```suggestion
  private static final CompactionServiceId DEFAULT_SERVICE_ID=CompactionServiceId.of("default");
  private CompactionServiceId service = DEFAULT_SERVICE_ID;
```

Reply:
I don't see why we need to have a hard-coded default name at all. Why can't this just always be a required parameter provided by the dispatcher?

Reply:
That is kinda what I was going for by putting the name of "default" set in the `SimpleCompactionDispatcher`

Reply:
> I don't see why we need to have a hard-coded default name at all. Why can't this just always be a required parameter provided by the dispatcher?

My thinking was that when the builder builds a default obj that it reflects the defaults in config.